module/VuFind/src/VuFind/Controller/AjaxController.php

```diff
@@ -361,6 +361,82 @@ protected function reduceServices(array $services)
     }
 
     /**
+     * Get Item Statuses
+     *
+     * This is responsible for printing the holdings information for a
+     * collection of records in JSON format.
+     *
+     * @return \Zend\Http\Response
+     * @author Chris Delis <cedelis@uillinois.edu>
+     * @author Tuan Nguyen <tuan@yorku.ca>
+     */
+    protected function getSingleItemStatusAjax()
```

Comment:
This seems like a lot of duplicated code for what should just be an edge case of the existing logic. Do we really need this? Can't we just send an array containing one ID to the existing logic?